Dockerfile

```diff
@@ -4,21 +4,23 @@ ARG USER_ID=1000
 ARG GROUP_ID=100
 ARG USER_NAME=foo
 
-RUN zypper -n install openssh-clients ansible ruby ruby-devel bash git libxml2-devel gcc make shadow vim curl
+RUN zypper ar -f -G https://download.opensuse.org/repositories/devel:/languages:/ruby/openSUSE_Leap_15.3/devel:languages:ruby.repo
+RUN zypper --gpg-auto-import-keys refresh
+
+RUN zypper -n install openssh-clients ansible ruby3.1 ruby3.1-devel bash git libxml2-devel gcc make shadow vim curl libxslt-devel
```

Comment:
*nitpick* `ruby3.1-devel` pulls in `ruby3.1`...

Reply:
yes, I know, I however always pull both to make sure to be explicit here. I can change however without problem :+1: